libs/perun/pipes/src/lib/is-all-selected.pipe.ts

```diff
@@ -0,0 +1,18 @@
+import { MatTableDataSource } from '@angular/material/table';
+import { TableCheckbox } from '@perun-web-apps/perun/services';
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'isAllSelected',
+})
+export class IsAllSelectedPipe implements PipeTransform {
+  constructor(private tableCheckboxService: TableCheckbox) {}
+
+  transform(
+    ds: MatTableDataSource<unknown>,
+    numberSelected: number,
+    canBeSelected: (T) => boolean
```

Comment:
Make this param optional, so you can skip it in HTML files, where this method does not exist.
```suggestion
    canBeSelected?: (T) => boolean
```